content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,248 @@
+
+Conversely, it might be more convenient to inject the `DD_*` environment variables at deploy time. If so, you can leverage the relevant API for your containerized infrastructure.
+
+**Note**: It's recommend to set **all three environment variables at the same time and place** if possible. This helps to control consistency across configuration files for all of your applications.
+
+{{< tabs >}}
+{{% tab "Kubernetes" %}}
+
+##### Pod-level metrics
+Add the following standard labels (`tags.datadoghq.com`) to the pod spec of a Deployment, StatefulSet, or Job.
+
+```yaml
+  template:
+    metadata:
+      labels:
+        tags.datadoghq.com/env: <ENV>
+        tags.datadoghq.com/service: <SERVICE>
+        tags.datadoghq.com/version: <VERSION>
+```
+These labels will cover pod-level Kubernetes CPU/Memory/Network/Disk metrics and will be used for injecting `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` into your service's container.
+
+If you have multiple containers per pod, you can specify standard labels by container:
+
+```yaml
+tags.datadoghq.com/<container-name>.env
+tags.datadoghq.com/<container-name>.service
+tags.datadoghq.com/<container-name>.version
+```
+
+##### State metrics
+
+For [Kubernetes State Metrics][1], add the same standard labels to the collection of labels for the parent resource (e.g., Deployment).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    tags.datadoghq.com/env: <ENV>
+    tags.datadoghq.com/service: <SERVICE>
+    tags.datadoghq.com/version: <VERSION>
+spec:
+  <Block from above>
+```
+
+##### APM Tracer / Statsd client
+
+[APM Tracer][2] and [statsd client][3] environment variables can be sourced from the same labels by using the [Kubernetes's downward API][4] in the format below:
+
+```yaml
+    containers:
+    -  ...
+       env:
+            - name: DD_ENV
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/env']
+            - name: DD_SERVICE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/service']
+            - name: DD_VERSION
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/version']
+```
+
+[1]: /agent/kubernetes/data_collected/#kube-state-metrics
+[2]: /tracing/send_traces/
+[3]: /integrations/statsd/
+[4]: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api
+{{% /tab %}}
+
+{{% tab "Docker" %}}
+To achieve a single point of configuration across all telemetry, make `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` available within your container's runtime environment. As long as the agent can access your container's [Docker socket][1], it will be able to detect the environment variables and map them to the standard tags.
```

Comment:
```suggestion
To achieve a single point of configuration across all telemetry, make `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` available within your container's runtime environment. As long as the Agent can access your container's [Docker socket][1], it will be able to detect the environment variables and map them to the standard tags.
```